app/src/main/java/org/xbmc/kore/ui/sections/file/MediaFileListFragment.java

```diff
@@ -323,7 +324,7 @@ private void browseDirectory(final FileLocation dir) {
 
         Files.GetDirectory action = new Files.GetDirectory(dir.file,
                 mediaType,
-                new ListType.Sort(ListType.Sort.SORT_METHOD_LABEL, true, true),
+                new ListType.Sort(ListType.Sort.SORT_METHOD_PATH, true, true),
```

Comment:
Ideally this would be an option on the overflow menu, like it is for instance in the movies section. Nvertheless, if you're not confortable implementing it, leave it and i'll merge it as is.